### PR TITLE
updated logic cask

### PR DESCRIPTION
**Pull request by guyzmo**
updated `logic` cask to a newer version that support the newer Saleae devices, cf the [changelog](http://support.saleae.com/hc/en-us/articles/201589175?flash_digest=356c7198f8123de41cd1d40a8d3fd5e3ebe0b8a2). It's marked as beta, but is recommended by the Saleae team.

Signed-off-by: Guyzmo <guyzmo+github@m0g.net>

### CODE REVIEW

File: Casks/logic.rb [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-  homepage 'http://www.saleae.com/'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  url "http://downloads.saleae.com/betas/#{version}/Logic-#{version}-Darwin.dmg"
 
-  pkg "Logic #{version} (10.5+).pkg"
+  homepage 'http://www.saleae.com/'
+  license :commercial
 
-  uninstall :pkgutil => 'com.saleae.saleae.Logic.pkg'
+  app "Logic.app"
```
Comment (vitorgalvao) [inline Casks/logic.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

**Comment by vitorgalvao** [comment]
> Signed-off-by: Guyzmo

You don’t need to keep adding these to the commits. It’s not a practice we follow.

**Comment by guyzmo** [comment]
I'm sorry for the signed-off, but I have the habit to do `git ci -s`, as in some other projects I commit in, they must be set… so I'll try not to do it, but habits die hard ;-)

**Comment by ndr-qef** [comment]
Thanks, merged as 493d683.

I would not worry about changing your git workflow to accomodate us; `--signoff` is really of no concern.

**Comment by vitorgalvao** [comment]
Yes, I really meant you don’t *need* to do it. No harm done if you do.

**Pull request closed without merging**